### PR TITLE
[bitnami/several] Fix default values and regenerate README

### PR DESCRIPTION
**Pull request by pablogalegoc**
**Description of the change**

Fix default values in _values.yaml_ and regenerate README using the automated tool

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Review by Mauraza — APPROVED** [review]
Hi @pablogalegoc,

Thanks for the PR, could you resolve the conflict? LGTM!!

**Review by carrodher — COMMENTED** [review]
Some minor topics

File: bitnami/concourse/values.yaml [hunk 1]
```diff
@@ -1083,7 +1083,7 @@ ingress:
   ## Either `hosts` or `rulesOverride` must be provided if Ingress is enabled.
   ## `rulesOverride` allows the user to define the full set of ingress rules, for more complex Ingress setups.
   ##
-  rulesOverride:
+  rulesOverride: []
```
[the remark this answers is not in the document]
Comment (pablogalegoc) [reply bitnami/concourse/values.yaml:1086]: In this case it should be a an array of objects, if I set a map I get:
`invalid type for io.k8s.api.networking.v1.IngressSpec.rules: got "map", expected "array"`

**Review by carrodher — APPROVED** [review]

**Comment by carrodher** [comment]
You need to regenerate the README once applied the above commits, apart from that LGTM

**Pull request merged**